sw/host/opentitantool/src/command/load_bitstream.rs

```diff
@@ -30,7 +30,7 @@ pub struct LoadBitstream {
     pub rom_kind: Option<RomKind>,
     #[structopt(long, parse(try_from_str=humantime::parse_duration), default_value="50ms", help = "Duration of the reset pulse.")]
     pub rom_reset_pulse: Duration,
-    #[structopt(long, parse(try_from_str=humantime::parse_duration), default_value="1s", help = "Duration of ROM detection timeout")]
+    #[structopt(long, parse(try_from_str=humantime::parse_duration), default_value="2s", help = "Duration of ROM detection timeout")]
```

Comment:
The timeout here appears to be necessary and insufficient to fix the issue on it's own, Without an increased timeout threshold We can see tests fail as I make these other changes.

Reply:
Without a 2s timeout:
```
//sw/device/tests:clkmgr_smoketest_fpga_cw310                            FAILED in 3.8s
  /usr/local/google/home/drewmacrae/.cache/bazel/_bazel_drewmacrae/eb3e804793dbbd45e50210cd3a3f84f1/execroot/lowrisc_opentitan/bazel-out/k8-fastbuild/testlogs/sw/device/tests/clkmgr_smoketest_fpga_cw310/test.log
//sw/device/tests:rstmgr_sw_req_test_fpga_cw310                          FAILED in 3.7s
  /usr/local/google/home/drewmacrae/.cache/bazel/_bazel_drewmacrae/eb3e804793dbbd45e50210cd3a3f84f1/execroot/lowrisc_opentitan/bazel-out/k8-fastbuild/testlogs/sw/device/tests/rstmgr_sw_req_test_fpga_cw310/test.log

Executed 137 out of 137 tests: 135 tests pass and 2 fail locally.
```

Reply:
I've created https://github.com/lowRISC/opentitan/issues/13195 to track that issue, because I'm not sure if it represents a way for our CI to get stuck on a bad bitstream.

Reply:
I can still hit it with bazel test --test_tag_filters=cw310,-broken --build_tests_only //... --runs_per_test=2 --runs_per_test_detects_flakes if I set the timeout to 1s